api/src/pcapi/routes/serialization/offers_serialize.py

```diff
@@ -249,11 +251,13 @@ class ListOffersOfferResponseModel(BaseModel):
     status: OfferStatus
     isShowcase: bool | None
     address: AddressResponseIsLinkedToVenueModel | None
+    is_headline_offer: bool
```

Comment:
ça devrait effectivement être en _snake_case_ ici, mais comme tout le reste est en _camelCase_, tu peux garder cette syntaxe. Ça rajoute de la dette sur du legacy , mais c'est mieux de garder de l'homogénéité. Ça te permet aussi de ne pas passer par le `getterdict`